CP1404_an/Practical_06/used_cars.py

```diff
@@ -0,0 +1,23 @@
+"""
+CP1404/CP5632 Practical - Client code to use the Car class.
+Note that the import has a folder (module) in it.
+This is why we name our folders with no spaces or capitals, as valid module names.
+"""
+
+from Practical_06.car import Car
+
+
+def main():
+    """Demo test code to show how to use car class."""
+    my_car = Car(180, "my_car")
+    my_car.drive(30)
+    print(f"Car has fuel: {my_car.fuel}")
+    print(my_car)
+
+    limo = Car(100, "limo")
```

Comment:
Same goes for this as well. Just like my comment for line 12, key first then value.